index.html

```diff
@@ -27,7 +27,7 @@
         <a href = "shaders/index.html?data=template.frag">Test Red</a>
       </div>
       <div class="col">
-        <a href = "shaders/index.html?data=texture_test.frag">Texture Test</a>
```

Comment:
If you have tested it and it works, lets use a name that does not contain the word test, but that tells that this example is using a texture. Maybe template_texture.frag?
Is it necessary that the the .frag file lies in the same folder where the jpg image is? if a new folder creation is necessary, same as above, i would not called it "test" ;)